tsMuxer/hevc.cpp

```diff
@@ -807,6 +809,74 @@ int HevcPpsUnit::deserialize()
     }
 }
 
+// ----------------------- HevcSeiUnit ------------------------
+HevcSeiUnit::HevcSeiUnit():
+isHDR10(false), isHDR10plus(false), isDV(false) {}
+
+int HevcSeiUnit::deserialize()
+{
+	int rez = HevcUnit::deserialize();
+	if (rez)
+		return rez;
+	try
+	{
+		int payloadType;
+		int payloadSize;
+		int nbyte;
+		do {
+			payloadType = 0;
+			payloadSize = 0;
+			nbyte = 0xFF;

```

Comment:
Any reason why these variables are initialised later (unconditionally, since the `do` block is going to be entered either way) instead of at declaration time? If not, just initialise them while declaring.

Reply:
@xavery once again, so true... Thanks for correcting.